staging/src/k8s.io/cluster-bootstrap/token/util/helpers.go

```diff
@@ -131,3 +134,80 @@ func ValidateUsages(usages []string) error {
 	}
 	return nil
 }
+
+// GetSecretString returns the string value for the given key in the specified Secret
+// If there is an error or if the key doesn't exist, an empty string is returned.
+func GetSecretString(secret *v1.Secret, key string) string {
```

Comment:
I suggest putting the secret functions in `util/secrets/secrets.go` or something.